rpc-server/src/modules/queries/methods.rs

```diff
@@ -250,22 +250,24 @@ async fn view_code(
         is_optimistic
     );
     let (code, account) = if is_optimistic {
-        tokio::try_join!(
+        futures::future::join(
```

Comment:
We don't have to wait for both tasks to be completed. The failure of any task should return a error immediately.